src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Report;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.service.CsvReader;
+import core.basesyntax.service.CsvWriter;
+import core.basesyntax.service.Mapper;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.CsvReaderImpl;
+import core.basesyntax.service.impl.CsvWriterImpl;
+import core.basesyntax.service.impl.FruitTransactionMapper;
+import core.basesyntax.service.impl.ReportMapper;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvReader reader = new CsvReaderImpl();
+        List<String> strings = reader.readLines("src/main/resources/fruits.csv");
```

Comment:
The relative path should be fetched from the resources folder in a more flexible way, using class loaders or resource streams, to avoid issues with different file systems and to comply with good practices of resource handling.